services/users-service/src/handlers/createUser.js

```diff
@@ -32,11 +39,33 @@ async function createUser(event, context) {
 
     await dynamoDB.put(params).promise();
 
-    return successResponse({ message: 'User created!' });
+    //OLD
+    // return successResponse({ message: 'User created!' });
+    return successResponse(user);
   } catch (error) {
     console.log(error);
     throw error;
   }
 }
 
 export const handler = commonMiddleware(createUser);
+
+function getStocks(numSelection, companies) {
+  const copy = {...companies};
+  for (let i = 0; i < numSelection; i++) {
+    let stockByCompany = getRandomValueDestructively(copy);
+
+    // unsure what the limit on stocks should be
+    stockByCompany.numStocks = getRandomIntZeroToX(20);
+    stocks.push(stockByCompany);
+  }
+}
+
+function calcCashOnHand(stocks) {
+  let total = 0;
+  for (let stock of stocks) {
```

Comment:
This is perfectly valid. Often in these cases (where you want to iterate over every element in the array) you probably want to use Array.map or Array.reduce

For example:
`stocks.reduce((total, stock) => { return total + (stock.pricePerShare * stock. numStocks) }, 0)`